src/mlpack/methods/ann/layer/linear.hpp

```diff
@@ -134,6 +134,13 @@ class Linear
   //! Modify the bias weights of the layer.
   OutputDataType& Bias() { return bias; }
 
+  //! Get the size of the weights.
+  size_t WeightSize() const
+  {
+    size_t weightSize = inSize * outSize;
+    return weightSize;
```

Comment:
```suggestion
    return inSize * outSize;
```

Just to keep it short.